closure/repositories.bzl

```diff
@@ -677,24 +677,23 @@ def com_google_jsinterop_annotations():
 def com_google_protobuf():
     http_archive(
         name = "com_google_protobuf",
-        strip_prefix = "protobuf-3.6.1.2",
-        sha256 = "2244b0308846bb22b4ff0bcc675e99290ff9f1115553ae9671eba1030af31bc0",
+        strip_prefix = "protobuf-3.6.1.3",
+        sha256 = "73fdad358857e120fd0fa19e071a96e15c0f23bb25f85d3f7009abfd4f264a2a",
         urls = [
-            "https://github.com/protocolbuffers/protobuf/archive/v3.6.1.2.tar.gz",
+            "https://github.com/protocolbuffers/protobuf/archive/v3.6.1.3.tar.gz",
+            "https://mirror.bazel.build/github.com/google/protobuf/archive/v3.6.1.3.tar.gz",
         ],
     )
 
 def com_google_protobuf_js():
     http_archive(
         name = "com_google_protobuf_js",
+        strip_prefix = "protobuf-3.6.1.3/js",
         urls = [
-            "https://mirror.bazel.build/github.com/google/protobuf/archive/v3.6.1.2.tar.gz",
-            "https://github.com/protocolbuffers/protobuf/archive/v3.6.1.2.tar.gz",
+            "https://github.com/protocolbuffers/protobuf/archive/v3.6.1.3.tar.gz",
+            "https://mirror.bazel.build/github.com/google/protobuf/archive/v3.6.1.3.tar.gz",
```

Comment:
Nit: Maybe you can set bazel mirror url as default by moving it before the github url?

Reply:
Can it be, that Buildifier would sort that list alphabetically anyway?

Reply:
Oh, that I am not very sure. @laurentlb Does buildifier sort the `urls` attribute?

Reply:
We don't lists in bzl files.